test/index.js

```diff
@@ -74,7 +74,6 @@ describe('PWMetrics', () => {
       it('should not call methods for calculating median results', () => {
         return pwMetrics.start().then(_ => {
           expect(findMedianRunStub).to.not.have.been.called;
-          expect(displayOutputStub).to.not.have.been.called;
```

Comment:
I did this because when only 1 run is specfied the code should display the graph. 
This change matches that behaviour, see https://github.com/paulirish/pwmetrics/pull/71/files#diff-6d186b954a58d5bb740f73d84fe39073L133